CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for releasing gateway EUI after deletion.
 - Added support in the Application Server for the `X-Downlink-Apikey`, `X-Downlink-Push` and `X-Downlink-Replace` webhook headers. They allow webhook integrations to determine which endpoints to use for downlink queue operations.
 - Added the `as.webhooks.downlinks.public-address` and `as.webhooks.downlinks.public-tls-address` configuration options to the Application Server. 
+- `--external-join-server` flag to `end-device update` CLI subcommand, which disassociates and deletes the device from cluster-local Join Server.
```

Comment:
Added ...

Reply:
And use the "official" form of the command instead of the alias (`end-devices update`; see `--help`)

Reply:
At https://keepachangelog.com/en/1.0.0/ these are not preceeded by `Added` and that was also the format we used so far